contracts/RiskManagerV1.sol

```diff
@@ -98,6 +98,12 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
     ISignerBondsSwapStrategy public newSignerBondsSwapStrategy;
     uint256 public signerBondsSwapStrategyInitiated;
 
+    // Amount of underwriter tokens granted to the notifier as reward for
+    // `notifyLiquidation` and `notifyLiquidated` calls.
+    uint256 public notifierReward;
+    uint256 public newNotifierReward;
+    uint256 public notifierRewardChangeInitiated;
```

Comment:
Can we put them after `auctionLengthChangeInitiated` so that all `uint256` fields are grouped together?

Reply:
I think it will be no longer applicable due to https://github.com/keep-network/coverage-pools/pull/111#discussion_r656862995